picard/util/tags.py

```diff
@@ -98,6 +177,46 @@ def script_name(self):
         else:
             return self._name
 
+    def notes(self):
+        for attrib, note in ATTRIB2NOTE.items():
+            if getattr(self, attrib):
+                yield note
```

Comment:
Should this be `yield _(note)`?

Reply:
Yes, absolutely, good catch.

It also shows we lack some tests.

I fixed this issue and added tests in https://github.com/rdswift/picard/pull/22